docs/configs/site.js

```diff
@@ -45,7 +45,7 @@ export default {
         children: [
           {
             key: 'docs0',
-            text: 'latest(3.1.1)',
+            text: 'latest(3.1.2)',
```

Comment:
we should change the file in dev version because only dev version file is work. But I will still merge it

Reply:
OK